### PR TITLE
Footer added

### PR DESCRIPTION
**Pull request by nvescio**
Summary

 - Footer

Screenshots
Chrome
footer

Trello Card
https://trello.com/c/z65SGK2B/14-footer

### CODE REVIEW

File: training_maquetado/index.html [hunk 1]
```diff
@@ -99,6 +99,12 @@ <h2 class="tech-release">Release: 2005</h2>
 
   </div>    
 
+  <div class="footer-container">
```
Comment (CarlosHWolox) [inline training_maquetado/index.html]: you can use the footer tag

File: training_maquetado/index.html [hunk 2]
```diff
@@ -99,6 +99,12 @@ <h2 class="tech-release">Release: 2005</h2>
 
   </div>    
 
+  <div class="footer-container"> 
+    <h2 class="footer-text">WE ARE GAME CHANGERS</h2>
+    <img class="logo-white" src="assets/wolox_logo_white.svg">
```
Comment (CarlosHWolox) [inline training_maquetado/index.html]: use alt text

File: training_maquetado/index.html [hunk 3]
```diff
@@ -99,6 +99,12 @@ <h2 class="tech-release">Release: 2005</h2>
...
+    <h6 class="copyright-text">Copyright 2018. Wolox. All rights reserved.</h6>
```
Comment (CarlosHWolox) [inline training_maquetado/index.html]: i wpould ise a p as it is not relevant info for seo spiders

File: training_maquetado/scss/footer.scss [hunk 4]
```diff
@@ -0,0 +1,21 @@
+.footer-container {
+  background-color: $blue;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 2% 15% 1%;
```
Comment (CarlosHWolox) [inline training_maquetado/scss/footer.scss]: use px

File: training_maquetado/scss/footer.scss [hunk 5]
```diff
@@ -0,0 +1,21 @@
+.footer-container {
+  background-color: $blue;
+  width: 100%;
```
Comment (CarlosHWolox) [inline training_maquetado/scss/footer.scss]: i think this is unnecessary

File: training_maquetado/scss/footer.scss [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+  color: white;
+  font-size: $text-big;
+  margin: 10px 0;
+}
+
+.copyright-text {
+  align-self: flex-end;
+  color: white;
+  font-size: $text-small;
+}
```
Comment (CarlosHWolox) [inline training_maquetado/scss/footer.scss]: Enter

File: training_maquetado/scss/footer.scss [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+  align-items: center;
+  padding: 2% 15% 1%;
+}
+
+.footer-text { 
+  color: white;
+  font-size: $text-big;
+  margin: 10px 0;
+}
+
```
Comment (CarlosHWolox) [inline training_maquetado/scss/footer.scss:15]: I would define a min-width for the image

File: training_maquetado/index.html [hunk 8]
```diff
@@ -99,6 +99,12 @@ <h2 class="tech-release">Release: 2005</h2>
 
   </div>    
 
+  <footer class="footer-container"> 
+    <h2 class="footer-text">WE ARE GAME CHANGERS</h2>
+    <img class="logo-white" alt="Wolox white logo" src="assets/wolox_logo_white.svg"/>
+    <p class="copyright-text">Copyright 2018. Wolox. All rights reserved.</p>
+  </footer>  
+
```
Comment (FrankIglesias) [inline training_maquetado/index.html]: Remove this line

File: training_maquetado/scss/footer.scss [hunk 9]
```diff
@@ -0,0 +1,24 @@
...
+.logo-white {
+  min-width: 400px;
+}
+
+.copyright-text {
+  align-self: flex-end;
+  color: white;
```
Comment (FrankIglesias) [inline training_maquetado/scss/footer.scss]: use color variables

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: training_maquetado/scss/colors.scss [hunk 10]
```diff
@@ -1,3 +1,4 @@
 $blue: #00ADEE;
 $green: #BED23C;
 $grey: #C4C4C4;
+$white: #FFFFFF;
```
Comment (FrankIglesias) [inline training_maquetado/scss/colors.scss]: this should be  written like #FFF

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**